examples/server.py

```diff
@@ -12,6 +12,7 @@
 
 def receive_requests():
     configure_opentelemetry(
+        span_exporter_endpoint="https://ingest.lightstep.com:443",
```

Comment:
isn't this the default endpoint? is the arg required?

Reply:
Removed :+1: